packages/sdk-socket-server-next/src/index.ts

```diff
@@ -49,8 +49,7 @@ configureSocketServer(server)
     // Make sure to protect the endpoint to be only available within the cluster for prometheus
     app.get('/metrics', async (_req, res) => {
       res.set('Content-Type', 'text/plain');
-      const metrics = extractMetrics({ ioServer });
-      res.send(metrics);
+      res.send(await read());
```

Comment:
No big deal but I think it could make sense to rename "read" to "readMetrics" or something to make it easier to understand what it does exactly